### ISSUE TITLE
Missing support in OPENAI_ORGANIZATION

### ISSUE THREAD

**Issue by barakb** [hwchase17/langchainjs#1495]
At the python lib it is present.
That make open AI unusable when one is part of an organization after end of the personal subscription period

**Comment by DLeddy** [comment]
Agreed. Pretty big blocker when there are prod and dev org_id's to toggle. Would be awesome if this was updated.

**Comment by JasperAlexander** [comment]
Agree!

**Comment by AlejandroDaneri** [comment]
Agree!

**Comment by kenzik** [comment]
No traction on this? :
